picard/ui/ui_options_maintenance.py

```diff
@@ -93,5 +125,7 @@ def retranslateUi(self, MaintenanceOptionsPage):
         self.open_folder_button.setText(_("Open folder"))
         self.load_backup_button.setText(_("Load Backup"))
         self.save_backup_button.setText(_("Save Backup"))
+        self.label_2.setText(_("Automatic backups directory"))
```

Comment:
> I might be misunderstanding, but I find "Automatic" misleading unless the backups are happening automatically.

I just thought it through again, and it dawned on me that you **are** creating backups automatically when loading a previously saved backup.  I was confused because I had assumed that it automatically created a backup when saving using the `Make It So!` button.  In one of my commercial products, I save a backup of the configuration file each time it is modified and saved (and only retain the most recent number specified by the user in the configuration settings), so that's what I was originally thinking.

Reply:
> Should we keep the previous title??

Reverted anyway as it shouldn't be done in this file anyway

Reply:
> I thought you were aware of it, isn't that your code?

Yes, but I was thinking that the setting label suggested that a backup was being made automatically whenever the configuration settings were updated using `Make It So!`.

> Should we keep the previous title??

I think it's fine the way you have it now.

Reply:
I see that you reverted the change to the label.  I guess we can just clarify it in the documentation instead.  I still think the label should end with a colon though.  :wink: